package.json

```diff
@@ -19,6 +19,7 @@
     "precommit": "npm run lint",
     "snapshot": "jest --updateSnapshot",
     "start": "NODE_ENV=development BABEL_ENV=development webpack-dev-server --config=webpack/webpack.dev.config.js --progress",
+    "start:stage": "NODE_ENV=development BABEL_ENV=development webpack-dev-server --config=webpack/webpack.dev-stage.config.js --progress --disable-host-check",
```

Comment:
Commenting after merge, but you can get past the unsecured warning by passing `--https` here. It will configure a self-signed cert for your dev server.

Reply:
Get out of town! Will add that when I fix the typo in `/edx/hosts` in the readme.